lib/resources/content/tsconfig.json

```diff
@@ -10,7 +10,13 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "moduleResolution": "node",
-    "lib": ["es2017", "dom"]
+    "lib": ["es2017", "dom"],
+    "baseUrl": ".",
+    "paths": {
+      "*": [
+        "./src/*"
```

Comment:
Not a typescript guru either, so forgive me as well. The TypeScript [path mapping docs](https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping) has some info on this. This line is matching everything to src, yep. I think it should include matching `'*'` as shown in the docs. I'm missing something here, appreciate your help.